interface-definitions/snmp.xml.in

```diff
@@ -123,18 +123,35 @@
           </leafNode>
           <leafNode name="oid-enable">
             <properties>
-              <help>Enable specific OIDs</help>
+              <help>Enable specific OIDs that by default are disable</help>
               <completionHelp>
-                <list>route-table</list>
+                <list>ip-route-table ip-net-to-media-table ip-net-to-physical-phys-address ip-cidr-route-table all</list>
               </completionHelp>
               <valueHelp>
-                <format>route-table</format>
-                <description>Enable routing table OIDs (ipCidrRouteTable inetCidrRouteTable)</description>
+                <format>all</format>
+                <description>Enable access to all oid</description>
```

Comment:
I would say that the `all` option can be omitted here. If someone wants to shoot into the leg - let's force him to load bullets one by one.